keps/sig-apps/0008-sidecarcontainer.md

```diff
@@ -0,0 +1,146 @@
+---
+kep-number: 0008
+title: Sidecar Containers
+authors:
+  - "@joseph-irving"
+owning-sig: sig-apps
+participating-sigs:
+  - sig-apps
+  - sig-node
+reviewers:
+  - TBD
```

Comment:
Need reviewers and approvers to merge

Reply:
Happy to be a reviewer. Need a sig-apps lead cod approver.

Reply:
I've added you as reviewer @fejta